tests/Api/GeneralTest.php

```diff
@@ -42,4 +44,36 @@ public function test_non_acceptable_content_type()
 
         $response->assertStatus(406);
     }
+
+    public function test_can_get_system_status()
+    {
+        $response = $this->json('GET', '/api/v1/status');
+
+        $response->assertStatus(200)
+                 ->assertHeader('Cache-Control')
+                 ->assertJsonFragment([
+                     'data' => [
+                         'status'  => 'success',
+                         'message' => 'System operational',
+                     ],
+                 ]);
+    }
+
+    public function test_can_get_system_status_not_success()
+    {
+        factory(Component::class)->create([
+            'status' => 3,
```

Comment:
Is there a constant you can use here, instead of the magic number `3`?

Reply:
Not currently. There will be added difficulty in this once we allow people to configure their own status values as well.